src/components/CircularOutline/CircularOutline.js

```diff
@@ -144,18 +141,14 @@ class RoundedOutline extends Component<Props, State> {
                   innerRef={node => (this.shapeNode = node)}
                   x={0}
                   y={0}
-                  width={width}
-                  height={height}
+                  width="100%"
+                  height="100%"
                   rx={height / 2}
-                  ry={width / 2}
+                  ry={height / 2}
                   fill="none"
                   stroke={`url(#${svgId})`}
                   strokeWidth={strokeWidth}
                   strokeLinecap="round"
-                  style={{
-                    strokeDasharray: `${pathLength}, ${pathLength + 1}`,
-                    strokeDashoffset: interpolatedDashOffset,
-                  }}
```

Comment:
This styling is required for the animation shown below to work.

![animation](https://user-images.githubusercontent.com/18172185/45239471-66e56580-b2b3-11e8-9f13-f59a75590b8e.gif)

Reply:
I have now put the stroke dash array code back in